ansible/roles/docker-tools/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Install additional docker tools globally
+
+# Install as many python dependencies using RPMs if possible to reduce risk
+# of future conflicts
+- name: docker | Install docker python tools
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - python-devel
+  - python-pip
+  - PyYAML
+  - python-six
+  - python-requests
+
+# TODO: Ansible fails with latest version of docker-py
+# Error: docker-py version is 1.10.2. Minimum version required is 1.7.0.
+# See https://github.com/ansible/ansible/issues/17495
+#
+# The latest docker-compose requires docker-py==1.10.3, so cap version
+# https://github.com/docker/compose/blob/1.8.1/requirements.txt#L4
+- name: docker swarm | install python packages
+  become: yes
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - docker-py==1.9.0
```

Comment:
and pip must be upgraded as well, see https://github.com/aleksandra-tarkowska/infrastructure/blob/3b7c0862a0388a867c1b58a36170e291b96bdddf/ansible/roles/devspace/tasks/main.yml#L41